penguin/app/components/welcome.js

```diff
@@ -24,9 +26,17 @@ class Welcome extends Component{
   }
 
   render(){
+    var imageLink = `${IP_address}/users/${this.props.username}/profilePhoto`;
+    console.log('match.js link to image: ', imageLink);
     return (
       <View style={styles.mainContainer}>
         <Text style={styles.welcomeText}>welcome, {this.props.firstName || 'rando'}</Text>
+        <Image 
+          style={styles.avatar} 
```

Comment:
@michelleheh Solid. I like the photo. 

I think the username, fun fact, and email seem a bit out of place here for some reason. Maybe another group could add in the ability to edit and view profile contents and account settings?  It's not bad so I wouldn't mind keeping it but I'm not sure if knowing your username, fun fact, and email bring any functional value.

What are your thoughts @nthaniel?

Attaching a picture ![](https://dl.dropboxusercontent.com/s/olv8ulfzjn02fkd/IMG_2183.PNG?dl=0)